Remotion/Development/UnitTesting.Compilation/Roslyn/CSharpRoslynCodeProvider.cs

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) RUBICON IT GmbH, www.rubicon.eu
```

Comment:
Please remind me: that's the least-code version to do this, correct? Aside from stripping out lines of code, we can't remove any of those Roslyn files.